.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
Prettier recommends auto-formatting on save using a tool like lint-staged vs. using a linter plugin. See https://prettier.io/docs/en/comparison.html for details.